colossalai/device/utils.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+
+
+def execute_cmd(cmd):
+    cmd = ' '.join(cmd)
+    print(cmd)
+    subprocess.call(cmd, shell=True)
+
+
+def store_tmp(time, bandwidth):
+    f = open("tmp.txt", "w")
```

Comment:
I don't recommend store the results in local files.

Reply:
You should keep it as data structure in memory or you can save it in a temp file with `import tempfile`.